### PR TITLE
Codegen for Azure Monitor

### PR DESCRIPTION
**Pull request by rakshith91**

### CODE REVIEW

File: sdk/monitor/opentelemetry-exporter-azuremonitor/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+## Getting started
+
+## Key concepts
+
+## Examples
+
+## Troubleshooting
+
+## Next steps
```
Comment (rakshith91) [inline sdk/monitor/opentelemetry-exporter-azuremonitor/README.md:11]:
```suggestion
## Next steps

## Contributing
```

File: eng/.docsettings.yml [hunk 2]
```diff
@@ -96,6 +96,7 @@ known_content_issues:
   - ['sdk/storage/azure-storage-queue/swagger/README.md', '#4554']
   - ['sdk/storage/README.md', '#4554']
   - ['sdk/textanalytics/azure-ai-textanalytics/samples/README.md', '#4554']
+  - ['sdk/monitor/opentelemetry-exporter-azuremonitor/swagger/README.md', '#4554']
```
Comment (rakshith91) [inline eng/.docsettings.yml:99]:
```suggestion
  - ['sdk/monitor/opentelemetry-exporter-azuremonitor/swagger/README.md', '#4554']
  - ['/sdk/monitor/opentelemetry-exporter-azuremonitor/README.md'. '#4554']
```

File: eng/.docsettings.yml [hunk 3]
```diff
@@ -96,6 +96,8 @@ known_content_issues:
...
+  - ['/sdk/monitor/opentelemetry-exporter-azuremonitor/README.md'. '#4554']
```
Comment (rakshith91) [inline eng/.docsettings.yml:100]:
```suggestion
  - ['/sdk/monitor/opentelemetry-exporter-azuremonitor/README.md', '#4554']
```

File: sdk/monitor/opentelemetry-exporter-azuremonitor/README.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Azure Monitor Opentelemtry Exporter
```
Comment (rakshith91) [inline sdk/monitor/opentelemetry-exporter-azuremonitor/README.md:1]:
```suggestion
# Azure Opentelemetry Exporter for Monitor
```

File: eng/.docsettings.yml [hunk 5]
```diff
@@ -96,6 +96,8 @@ known_content_issues:
   - ['sdk/storage/azure-storage-queue/swagger/README.md', '#4554']
   - ['sdk/storage/README.md', '#4554']
   - ['sdk/textanalytics/azure-ai-textanalytics/samples/README.md', '#4554']
+  - ['sdk/monitor/opentelemetry-exporter-azuremonitor/swagger/README.md', '#4554']
+  - ['/sdk/monitor/opentelemetry-exporter-azuremonitor/README.md', '#4554']
```
Comment (rakshith91) [inline eng/.docsettings.yml:100]:
```suggestion
  - ['sdk/monitor/opentelemetry-exporter-azuremonitor/README.md', '#4554']
```

File: sdk/monitor/opentelemetry-exporter-azuremonitor/setup.py [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+    long_description_content_type='text/markdown',
+    license='MIT License',
+    author='Microsoft Corporation',
+    author_email='ascl@microsoft.com',
+    url='https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/monitor/opentelemetry-exporter-azuremonitor',
+    classifiers=[
+        "Development Status :: 4 - Beta",
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
```
Comment (lmazuel) [inline sdk/monitor/opentelemetry-exporter-azuremonitor/setup.py:62]: Python 3 only no?

File: sdk/monitor/opentelemetry-exporter-azuremonitor/setup.py [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
```
Comment (lmazuel) [inline sdk/monitor/opentelemetry-exporter-azuremonitor/setup.py:67]: It's time to add 3.9 everywhere now

File: sdk/monitor/opentelemetry-exporter-azuremonitor/setup.py [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+        'License :: OSI Approved :: MIT License',
+    ],
+    zip_safe=False,
+    packages=find_packages(exclude=[
+        # Exclude packages that will be covered by PEP420 or nspkg
```
Comment (lmazuel) [inline sdk/monitor/opentelemetry-exporter-azuremonitor/setup.py:72]: That's a good question, we need to make sure we ship it in a way that the opentelemetry package expect, this is not necessarly our guidelines that matters here. Did you test it?

Comment (rakshith91) [reply sdk/monitor/opentelemetry-exporter-azuremonitor/setup.py:72]:
```
pip install -e .
pip install opentelemtry-sdk
```

```
>>> from opentelemetry import trace 
>>> from opentelemetry.exporter import azuremonitor
>>> from opentelemetry.exporter.azuremonitor import AzureMonitorExporter  
>>> from opentelemetry.sdk.trace import TracerProvider
>>> from opentelemetry.sdk.trace.export import BatchExportSpanProcessor
```

File: sdk/monitor/opentelemetry-exporter-azuremonitor/LICENSE.txt [hunk 9]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2017 Microsoft
```
Comment (hectorhdzg) [inline sdk/monitor/opentelemetry-exporter-azuremonitor/LICENSE.txt:3]:
```suggestion
Copyright (c) Microsoft Corporation.
```

**Review by hectorhdzg — APPROVED** [review]
LGTM

**Review by lmazuel — APPROVED** [review]

**Pull request merged**